### PR TITLE
XLA CMake integration

### PR DESCRIPTION
**Pull request by adanlopeza**
- Edited CMakeLists.txt to enable xla integration, if needed.

- Added automous XLA folder with its own CMakeLists.txt for advanced build setup.

### CODE REVIEW

File: xla/jfdctxla.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ * jfdctxla.c
+ *
+ * Copyright (C) 1994-1996, Thomas G. Lane.
```
Comment (itaiz-google) [inline xla/jfdctxla.c]: I think the copyright line should be changed to reference Google.

File: xla_none.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/*
+ * xla_none.c
+ *
+ * Copyright 2009 Pierre Ossman <ossman@cendio.se> for Cendio AB
```
Comment (itaiz-google) [inline xla_none.c]: Same comment re copyright.

File: xla_none.c [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/*
```
Comment (itaiz-google) [inline xla_none.c:1]: Do we need this file?  Was it included by accident?

Comment (adanlopeza) [reply xla_none.c:1]: We do need this file, otherwise the function isn't linked. SIMD uses the same strategy in jsimd_none.c.

**Pull request closed without merging**